<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLogin</class>
 <widget class="QDialog" name="DlgLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>570</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/IconResource/lightning.png</normaloff>:/IconResource/lightning.png</iconset>
  </property>
  <widget class="QLabel" name="labBkg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>501</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="bkgresource.qrc">:/BkgResource/bkg_login.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>610</width>
     <height>208</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layMian">
    <item>
     <layout class="QGridLayout" name="layInfo">
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtPass">
        <property name="styleSheet">
         <string notr="true">QLineEdit{color:black}
QLineEdit:hover{color:#F0F0F0}
QLineEdit{background-color:rgba(200,0,0,80)}
QLineEdit{border:2px}
QLineEdit{border-radius:10px}
QLineEdit{padding:2px 4px}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labUser">
        <property name="styleSheet">
         <string notr="true">QLabel{color:red}
QLabel{padding:2px 4px}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>edtUser</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labPass">
        <property name="styleSheet">
         <string notr="true">QLabel{color:red}
QLabel{padding:2px 4px}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>edtPass</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="chbPass">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{color:red}
QCheckBox:hover{color:black}
</string>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtUser">
        <property name="whatsThis">
         <string>fdghj</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{color:black}
QLineEdit:hover{color:#F0F0F0}
QLineEdit{background-color:rgba(200,0,0,80)}
QLineEdit{border:2px}
QLineEdit{border-radius:10px}
QLineEdit{padding:2px 4px}</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="chbAuto">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{color:red}
QCheckBox:hover{color:lightyellow}
</string>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chbAdmin">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{color:red}
QCheckBox:hover{color:black}
</string>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layOper">
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color:red}
QPushButton:hover{color:black}
QPushButton{background-color:rgba(221, 195, 47,75)}
QPushButton{border:2px}
QPushButton{border-radius:10px}
QPushButton{padding:2px 4px}
QPushButton{opacity:0.5}</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLogin">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color:red}
QPushButton:hover{color:black}
QPushButton{background-color:rgba(221, 195, 47,75)}
QPushButton{border:2px}
QPushButton{border-radius:10px}
QPushButton{padding:2px 4px}
QPushButton{opacity:0.5}</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFindPass">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color:red}
QPushButton:hover{color:black}
QPushButton{background-color:rgba(221, 195, 47,75)}
QPushButton{border:2px}
QPushButton{border-radius:10px}
QPushButton{padding:2px 4px}
QPushButton{opacity:0.5}</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>520</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color:brown}
QPushButton:hover{color:red}
QPushButton{border:2px}
QPushButton{border-radius:10px}
QPushButton{padding:2px 4px}
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtUser</tabstop>
  <tabstop>edtPass</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnFindPass</tabstop>
  <tabstop>chbAdmin</tabstop>
  <tabstop>chbPass</tabstop>
  <tabstop>chbAuto</tabstop>
 </tabstops>
 <resources>
  <include location="bkgresource.qrc"/>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
